homeassistant/components/telegram_bot/__init__.py

```diff
@@ -547,7 +547,7 @@ def _make_row_inline_keyboard(row_keyboard):
                 )
         return params
 
-    def _send_msg(self, func_send, msg_error, *args_msg, **kwargs_msg):
+    def _send_msg(self, func_send, msg_error, message_tag, *args_msg, **kwargs_msg):
```

Comment:
`message_tag` is not a Telegram API parameter, it's just an internal state for the component. Previously it was mixed with all the other params (that were passed to the Telegram API), now this is handled separately.